index.js

```diff
@@ -56,4 +62,20 @@ bookList.addEventListener('click', (event) => {
     Abooksa.displayBooks();
   }
 });
-Abooksa.displayBooks();
\ No newline at end of file
+Abooksa.displayBooks();
+
+menua.addEventListener('click', () => {
```

Comment:
- [x] [Optional] Your all functionality is working as expected😃 , but I see that when I remove the item, there is a bug with the remove functionality from the previous milestone : 
- Check the video loom [here]( https://www.loom.com/share/5dce290f71e242ab94910048d0c6596d) 😊⤵️for your project when I try to test it you will notice that when I add for instance three items and try to remove the second one I see that all the items removed from the page, not the specific item, so it will be better if you fix that too make your Awesome books complete for all functionality 💯😊

- Hint for external code to understand the removing functionality 😊⤵️: 

```snippet
removeTodo(todoID) {
    this.list = this.list.filter((todo) => todo.id !== todoID);
    this.list.forEach((todo, index) => {
      todo.index = index + 1;
    });
    const newTodos = this.list;
    localStorage.setItem('todos', JSON.stringify(newTodos));
  }
```